gramps/plugins/webreport/narrativeweb.py

```diff
@@ -2382,10 +2382,15 @@ def __add_advanced_options_2(self, menu):
 
         inc_places = BooleanOption(_('Include place pages'), False)
         inc_places.set_help(
-            _('Whether or not to include the place pages.'))
+            _('Whether or not to include the place Pages.'))
         addopt("inc_places", inc_places)
 
-        inc_sources = BooleanOption(_('Include source pages'), False)
+        inc_uplaces = BooleanOption(_('Include unused place pages'), False)
+        inc_uplaces.set_help(
+            _('Whether or not to include the unused place Pages.'))
+        addopt("inc_uplaces", inc_uplaces)
+
+        inc_sources = BooleanOption(_('Include sources pages'), False)
```

Comment:
This should be changed back to "source pages".

Reply:
I hope this is solved.